### ISSUE TITLE
SessionListItem UI updates for quizzes

### ISSUE THREAD

**Issue by ReckoningReckoner** [qlicker/qlicker#355]
- UI should update for students on the courses tab which indicates that a quiz, not a session is being run.
- Quizzes still appear red in professor course view when not under review but fully graded

In my opinion, the "allow review" button already gives an indication as to whether the session is reviewable or not. If more indication is needed, then clicking the "allow review" should change the state from "FINISHED" to something like "REVIEWABLE", and should accordingly be selectable in the session toolbar.

**Comment by ryanmartinneutrino** [comment]
Agree, it should be better. One issue is all of the buttons that we have whose text changes when you click on them, particularly in running session (e.g. hide/show stats). We need a good way to tell the user what the state of the button actually is. The intent here, by colouring the sessions that students cannot review, was to draw specific attention to those, to encourage profs to make them reviewable. 

I agree that colouring it red also signals that it needs grading, which it shouldn't, so I agree with your proposed changes. I think we can clean this all up when we improve SessionListItem to differentiate between a quiz and a  live session...
